libexec/bootstrap-scripts/deffile-sections.sh

```diff
@@ -167,7 +167,12 @@ if [ -z "${SINGULARITY_BUILDSECTION:-}" -o "${SINGULARITY_BUILDSECTION:-}" == "l
 
         singularity_section_get "labels" "$SINGULARITY_BUILDDEF" | while read KEY VAL; do
             if [ -n "$KEY" -a -n "$VAL" ]; then
-                $ADD_LABEL --key "$KEY" --value "$VAL" --file "$SINGULARITY_ROOTFS/.singularity.d/labels.json"
+                if [ "${SINGULARITY_DEFFILE_BOOTSTRAP:-}" = "shub" -o "${SINGULARITY_DEFFILE_BOOTSTRAP:-}" = "localimage" ]; then
+                    $ADD_LABEL --key "$KEY" --value "$VAL" --file "$SINGULARITY_ROOTFS/.singularity.d/labels.json" -f "False"
+                else
```

Comment:
Hey @GodloveD ! The force argument is a boolean, so you just need to add the flag (without any True/False), eg, here is the definition in python:
```
    parser.add_option('-f', dest="force",
                      help="force add (overwrite if exists)",
                      default=False, action='store_true')
```
and then it would just work like:

```
 $ADD_LABEL --key "$KEY" --value "$VAL" --file "$SINGULARITY_ROOTFS/.singularity.d/labels.json" -f
```

Reply:
Did you try it with the "True" and does it error? Or just ignore it?

Reply:
Of course you are right.  duh.  Thanks.

Reply:
haha no worries! There isn't clear documentation about this... it's all IN MY HEAD. :scream: